httpgrpc/README.md

```diff
@@ -0,0 +1,5 @@
+To rebuild generated protobuf code, run:
+
+    protoc -I ./ --go_out=plugins=grpc:./ ./httpgrpc.proto
```

Comment:
I think technically it's "vendee", but a more colloquial English translation would be "buyer".